diff -aurN beagle-0.2.7.orig/Util/Galago.cs beagle-0.2.7/Util/Galago.cs
--- beagle-0.2.7.orig/Util/Galago.cs	2005-08-14 19:54:17.000000000 +0200
+++ beagle-0.2.7/Util/Galago.cs	2006-08-13 08:56:37.000000000 +0200
@@ -35,15 +35,15 @@
 
 		public static string GetPresence (string service_id, string username)
 		{
-			if (! Galago.Core.Init ("beagle-galago-presence"))
+			if (! Galago.Global.Init ("beagle-galago-presence"))
 				return null;
 
-			Galago.Service service = Galago.Core.GetService (service_id, false, true);
+			Galago.Service service = Galago.Global.GetService (service_id);
 
 			if (service == null)
 				return null;
 
-			Galago.Account account = service.GetAccount (username, true);
+			Galago.Account account = service.GetAccount (username);
 
 			if (account == null)
 				return null;
@@ -56,14 +56,14 @@
 
 			string user_status = null;
 
-			if (presence.Idle == true) 
+			if (presence == null) 
+				user_status = ("No presence");
+			else
 				// FIXME: We need to translate this
 				user_status = String.Format ("Idle {0}",
-							     StringFu.DurationToPrettyString (DateTime.Now.AddSeconds (presence.IdleTime), DateTime.Now));
-			else
-				user_status = presence.ActiveStatus.Name;
+							     StringFu.DurationToPrettyString (presence.IdleTime, DateTime.Now));
 
-			Galago.Core.Uninit();
+			Galago.Global.Uninit();
 
 			return user_status;
 		}
