diff -urN beagle-0.3.8/conf-data/crawl-rules/crawl-applications beagle-0.3.8.new/conf-data/crawl-rules/crawl-applications
--- beagle-0.3.8/conf-data/crawl-rules/crawl-applications	2008-01-03 10:22:37.000000000 +0100
+++ beagle-0.3.8.new/conf-data/crawl-rules/crawl-applications	2008-07-27 17:42:37.000000000 +0200
@@ -2,7 +2,7 @@
 
 CRAWL_INDEX_NAME="applications"
 
-CRAWL_PATHS="/usr/share/applications /usr/local/share/applications /opt/gnome/share/applications /opt/kde3/share/applications"
+CRAWL_PATHS="/usr/share/applications /usr/local/share/applications"
 
 CRAWL_RECURSIVE="yes"
 
diff -urN beagle-0.3.8/conf-data/crawl-rules/crawl-documentation beagle-0.3.8.new/conf-data/crawl-rules/crawl-documentation
--- beagle-0.3.8/conf-data/crawl-rules/crawl-documentation	2008-01-03 10:22:37.000000000 +0100
+++ beagle-0.3.8.new/conf-data/crawl-rules/crawl-documentation	2008-07-27 17:45:43.000000000 +0200
@@ -2,7 +2,7 @@
 
 CRAWL_INDEX_NAME="documentation"
 
-CRAWL_PATHS="/usr/share/doc /usr/local/share/doc /opt/kde3/share/doc /opt/gnome/share/gnome/help /usr/share/gnome/help /opt/gnome/share/gtk-doc/html /usr/share/gtk-doc/html /usr/share/gnome/html"
+CRAWL_PATHS="/usr/share/doc /usr/local/share/doc /usr/share/gnome/help"
 
 CRAWL_ALLOW_PATTERNS="*.xml,*.html,*.docbook,*.pdf"
 
diff -urN beagle-0.3.8/conf-data/crawl-rules/crawl-executables beagle-0.3.8.new/conf-data/crawl-rules/crawl-executables
--- beagle-0.3.8/conf-data/crawl-rules/crawl-executables	2008-04-26 04:47:25.000000000 +0200
+++ beagle-0.3.8.new/conf-data/crawl-rules/crawl-executables	2008-07-27 17:46:08.000000000 +0200
@@ -3,7 +3,7 @@
 # Use the same index as the applications index
 CRAWL_INDEX_NAME="applications"
 
-CRAWL_PATHS="/bin /sbin /usr/bin /usr/sbin /usr/local/bin/ /usr/local/sbin"
+CRAWL_PATHS="/bin /sbin /usr/bin /usr/sbin /usr/local/bin /usr/local/sbin"
 
 # Only index files in the gives directories
 CRAWL_RECURSIVE="no"
diff -urN beagle-0.3.8/conf-data/crawl-rules/crawl-monodoc beagle-0.3.8.new/conf-data/crawl-rules/crawl-monodoc
--- beagle-0.3.8/conf-data/crawl-rules/crawl-monodoc	2008-01-03 10:22:37.000000000 +0100
+++ beagle-0.3.8.new/conf-data/crawl-rules/crawl-monodoc	2008-07-27 17:46:32.000000000 +0200
@@ -1,4 +1,4 @@
-CRAWL_ENABLED="no"
+CRAWL_ENABLED="yes"
 
 CRAWL_INDEX_NAME="monodoc"
 
diff -urN beagle-0.3.8/conf-data/crawl-rules/crawl-windows beagle-0.3.8.new/conf-data/crawl-rules/crawl-windows
--- beagle-0.3.8/conf-data/crawl-rules/crawl-windows	2008-01-03 10:22:37.000000000 +0100
+++ beagle-0.3.8.new/conf-data/crawl-rules/crawl-windows	2008-07-27 17:47:00.000000000 +0200
@@ -1,4 +1,4 @@
-CRAWL_ENABLED="yes"
+CRAWL_ENABLED="no"
 
 CRAWL_INDEX_NAME="windows"
 
