diff -urN beagle-0.2.16.2/configure.in beagle-0.2.16.2.new/configure.in
--- beagle-0.2.16.2/configure.in	2007-02-22 18:55:03.000000000 +0100
+++ beagle-0.2.16.2.new/configure.in	2007-03-14 21:14:59.000000000 +0100
@@ -480,7 +480,10 @@
 	enable_epiphany_extension=$enableval,
 	enable_epiphany_extension=auto)
 
-if $PKG_CONFIG --exists epiphany-2.16; then
+if $PKG_CONFIG --exists epiphany-2.18; then
+        EPIPHANY_MAJOR=2.18
+        EPIPHANY_REQUIRED=2.18.0
+elif $PKG_CONFIG --exists epiphany-2.16; then
         EPIPHANY_MAJOR=2.16
         EPIPHANY_REQUIRED=2.16.0
 elif $PKG_CONFIG --exists epiphany-2.14; then
