diff -aurN beagle-0.1.2.orig/Best/best.desktop beagle-0.1.2/Best/best.desktop
--- beagle-0.1.2.orig/Best/best.desktop	2005-11-10 21:22:35.000000000 +0100
+++ beagle-0.1.2/Best/best.desktop	2005-11-14 17:43:50.000000000 +0100
@@ -1,11 +1,11 @@
 [Desktop Entry]
-X-SuSE-translate=true
 Encoding=UTF-8
 Name=Beagle Search
 Name[en_CA]=Beagle Search
+Name[pl]=Wyszykiwanie Beagle
 GenericName=Desktop Search
 GenericName[en_CA]=Desktop Search
 Exec=best --no-tray
 Icon=best
 Type=Application
-Categories=GTK;GNOME;Application;Core;Utility;Filesystem;
+Categories=GTK;GNOME;Utility;
diff -aurN beagle-0.1.2.orig/tools/beagle-settings.desktop beagle-0.1.2/tools/beagle-settings.desktop
--- beagle-0.1.2.orig/tools/beagle-settings.desktop	2005-11-10 21:22:30.000000000 +0100
+++ beagle-0.1.2/tools/beagle-settings.desktop	2005-11-14 17:45:49.000000000 +0100
@@ -1,13 +1,14 @@
 [Desktop Entry]
 Encoding=UTF-8
-X-SuSE-translate=true
 Name=Search & Indexing
 Name[en_CA]=Search & Indexing
+Name[pl]=Wyszuiwanie i indekswanie
 GenericName=Configure Desktop Search & Indexing
 GenericName[en_CA]=Configure Desktop Search & Indexing
 Comment=Configure search & indexing properties
 Comment[en_CA]=Configure search & indexing properties
+Comment[pl]=Konfiguracja wyszukiwania i indeksowania
 Exec=beagle-settings
 Icon=gnome-searchtool
 Type=Application
-Categories=Application;Settings;AdvancedSettings;Utility;Filesystem;X-Ximian-Settings;
+Categories=GTK;GNOME;Settings;AdvancedSettings;
