--- beagle-0.2.18/configure.in~	2007-09-30 17:57:33.000000000 +0200
+++ beagle-0.2.18/configure.in	2007-09-30 17:58:01.000000000 +0200
@@ -479,7 +479,13 @@
 	enable_epiphany_extension=$enableval,
 	enable_epiphany_extension=auto)
 
-if $PKG_CONFIG --exists epiphany-2.16; then
+if $PKG_CONFIG --exists epiphany-2.20; then
+        EPIPHANY_MAJOR=2.20
+        EPIPHANY_REQUIRED=2.20.0
+elif $PKG_CONFIG --exists epiphany-2.18; then
+        EPIPHANY_MAJOR=2.18
+        EPIPHANY_REQUIRED=2.18.0
+elif $PKG_CONFIG --exists epiphany-2.16; then
         EPIPHANY_MAJOR=2.16
         EPIPHANY_REQUIRED=2.16.0
 elif $PKG_CONFIG --exists epiphany-2.14; then
