diff -urN beagle-0.3.8/configure.in beagle-0.3.8.new/configure.in
--- beagle-0.3.8/configure.in	2008-06-12 18:06:25.000000000 +0200
+++ beagle-0.3.8.new/configure.in	2008-07-27 00:39:56.000000000 +0200
@@ -123,12 +123,10 @@
 AC_FUNC_MMAP()
 
 # check for desktop-launch
-AC_PATH_PROG(DESKTOP_LAUNCH, desktop-launch, no)
-AM_CONDITIONAL(ENABLE_DESKTOP_LAUNCH, test "x$DESKTOP_LAUNCH" != "xno")
+AM_CONDITIONAL(ENABLE_DESKTOP_LAUNCH, false)
 
 # check for xdg-open
-AC_PATH_PROG(XDG_OPEN, xdg-open, no)
-AM_CONDITIONAL(ENABLE_XDG_OPEN, test "x$XDG_OPEN" != "xno")
+AM_CONDITIONAL(ENABLE_XDG_OPEN, true)
 
 #
 # I18N
