diff -ur beagle-0.3.3/beagled/beagled-autostart.desktop beagle-0.3.3-desktop/beagled/beagled-autostart.desktop
--- beagle-0.3.3/beagled/beagled-autostart.desktop	2007-11-27 02:50:05.000000000 +0100
+++ beagle-0.3.3-desktop/beagled/beagled-autostart.desktop	2008-02-03 17:34:30.000000000 +0100
@@ -1,8 +1,13 @@
 [Desktop Entry]
+Encoding=UTF-8
+Name=Beagle
+Comment=A system for searching and managing your personal information space
+Comment[pl]=System przeszukiwania informacji osobistych i zarządzania nimi
 Exec=beagled --replace
-Name=Beagle Search Daemon
+Icon=stock-init
 Terminal=false
 Type=Application
 StartupNotify=false
 X-KDE-autostart-phase=2
 X-KDE-autostart-after=panel
+X-GNOME-Autostart-enabled=true
diff -ur beagle-0.3.3/search/beagle-search.desktop.in beagle-0.3.3-desktop/search/beagle-search.desktop.in
--- beagle-0.3.3/search/beagle-search.desktop.in	2008-01-02 08:59:43.000000000 +0100
+++ beagle-0.3.3-desktop/search/beagle-search.desktop.in	2008-02-03 17:36:00.000000000 +0100
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Core;Utility;Filesystem;
+Categories=GNOME;GTK;Utility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=beagle
 X-GNOME-Bugzilla-Component=General
diff -ur beagle-0.3.3/search/beagle-search.desktop.in.in beagle-0.3.3-desktop/search/beagle-search.desktop.in.in
--- beagle-0.3.3/search/beagle-search.desktop.in.in	2007-11-27 02:49:58.000000000 +0100
+++ beagle-0.3.3-desktop/search/beagle-search.desktop.in.in	2008-02-03 17:35:49.000000000 +0100
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Core;Utility;Filesystem;
+Categories=GNOME;GTK;Utility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=beagle
 X-GNOME-Bugzilla-Component=General
diff -ur beagle-0.3.3/tools/beagle-settings.desktop.in.in beagle-0.3.3-desktop/tools/beagle-settings.desktop.in.in
--- beagle-0.3.3/tools/beagle-settings.desktop.in.in	2007-11-27 02:49:57.000000000 +0100
+++ beagle-0.3.3-desktop/tools/beagle-settings.desktop.in.in	2008-02-03 17:36:48.000000000 +0100
@@ -4,7 +4,7 @@
 Exec=beagle-settings
 Icon=system-search
 Type=Application
-Categories=Settings;Utility;Filesystem;X-Ximian-Settings;
+Categories=GTK;GNOME;Settings;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=beagle
 X-GNOME-Bugzilla-Component=General

