diff -ur beagle-0.2.15.1/beagled/beagled-autostart.desktop beagle-0.2.15.1-desktop/beagled/beagled-autostart.desktop
--- beagle-0.2.15.1/beagled/beagled-autostart.desktop	2007-01-24 18:52:36.000000000 +0100
+++ beagle-0.2.15.1-desktop/beagled/beagled-autostart.desktop	2007-01-27 09:24:41.000000000 +0100
@@ -1,8 +1,13 @@
 [Desktop Entry]
 Encoding=UTF-8
+Name=Beagle
+Comment=A system for searching and managing your personal information space
+Comment[pl]=System przeszukiwania informacji osobistych i zarządzania nimi
+Icon=stock_init
 Exec=beagled
-Name=Beagle Search Daemon
 Terminal=false
+Type=Application
 X-KDE-autostart-phase=2
 X-KDE-autostart-after=panel
 X-KDE-StartupNotify=false
+X-GNOME-Autostart-enabled=true
diff -ur beagle-0.2.15.1/search/beagle-search.desktop.in beagle-0.2.15.1-desktop/search/beagle-search.desktop.in
--- beagle-0.2.15.1/search/beagle-search.desktop.in	2007-01-26 00:39:00.000000000 +0100
+++ beagle-0.2.15.1-desktop/search/beagle-search.desktop.in	2007-01-27 09:24:41.000000000 +0100
@@ -1,5 +1,4 @@
 [Desktop Entry]
-X-SuSE-translate=true
 _Name=Search
 _GenericName=Search
 _Comment=Search for data on your desktop
@@ -9,7 +8,7 @@
 Type=Application
 StartupNotify=true
 Encoding=UTF-8
-Categories=GNOME;GTK;Application;Core;Utility;Filesystem;
+Categories=GTK;GNOME;Utility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=beagle
 X-GNOME-Bugzilla-Component=General
--- beagle-0.2.16/tools/beagle-settings.desktop.in~	2007-02-10 23:17:37.000000000 +0100
+++ beagle-0.2.16/tools/beagle-settings.desktop.in	2007-02-10 23:22:36.000000000 +0100
@@ -1,12 +1,11 @@
 [Desktop Entry]
 Encoding=UTF-8
-X-SuSE-translate=true
 _Name=Search & Indexing
 _GenericName=Configure Desktop Search
 Exec=beagle-settings
 Icon=system-search
 Type=Application
-Categories=Application;Settings;AdvancedSettings;Utility;Filesystem;X-Ximian-Settings;
+Categories=GTK;GNOME;Settings;AdvancedSettings;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=beagle
 X-GNOME-Bugzilla-Component=General
