diff -aurN beagle-0.2.7.orig/tools/crawl-rules/crawl-applications beagle-0.2.7/tools/crawl-rules/crawl-applications
--- beagle-0.2.7.orig/tools/crawl-rules/crawl-applications	2006-04-07 18:16:02.000000000 +0200
+++ beagle-0.2.7/tools/crawl-rules/crawl-applications	2006-06-27 20:24:01.295867816 +0200
@@ -2,7 +2,7 @@
 
 CRAWL_INDEX_NAME="applications"
 
-CRAWL_PATHS="/usr/share/applications /usr/local/share/applications /opt/gnome/share/applications /opt/kde3/share/applications"
+CRAWL_PATHS="/usr/share/applications"
 
 CRAWL_RECURSIVE="yes"
 
diff -aurN beagle-0.2.7.orig/tools/crawl-rules/crawl-documentation beagle-0.2.7/tools/crawl-rules/crawl-documentation
--- beagle-0.2.7.orig/tools/crawl-rules/crawl-documentation	2006-06-19 19:54:27.000000000 +0200
+++ beagle-0.2.7/tools/crawl-rules/crawl-documentation	2006-06-27 20:25:28.212654456 +0200
@@ -2,7 +2,7 @@
 
 CRAWL_INDEX_NAME="documentation"
 
-CRAWL_PATHS="/usr/share/doc /usr/local/share/doc /opt/kde3/share/doc /opt/gnome/share/gnome/help /usr/share/gnome/help /opt/gnome/share/gtk-doc/html /usr/share/gtk-doc/html /usr/share/gnome/html"
+CRAWL_PATHS="/usr/share/doc /usr/share/gnome/help"
 
 CRAWL_ALLOW_PATTERNS="*.xml,*.html,*.docbook"
 
diff -aurN beagle-0.2.7.orig/tools/Makefile.am beagle-0.2.7/tools/Makefile.am
--- beagle-0.2.7.orig/tools/Makefile.am	2006-06-19 19:54:27.000000000 +0200
+++ beagle-0.2.7/tools/Makefile.am	2006-06-27 20:25:56.806307560 +0200
@@ -35,7 +35,6 @@
 	chmod +x $(CRAWL_WRAPPER)
 
 CRAWL_RULES = \
-	$(srcdir)/crawl-rules/crawl-windows		\
 	$(srcdir)/crawl-rules/crawl-applications	\
 	$(srcdir)/crawl-rules/crawl-documentation
 
--- beagle-0.2.15.1/tools/beagle-crawl-system~	2007-01-24 18:56:19.000000000 +0100
+++ beagle-0.2.15.1/tools/beagle-crawl-system	2007-02-05 17:39:20.000000000 +0100
@@ -36,7 +36,7 @@
 MONO_SHARED_DIR=`mktemp -d -p $TMPDIR .beagleindexwapi.XXXXXXXXXX`|| ( echo "Can't create wapi directory!" ; exit 1 )
 chown $CRAWL_USER $MONO_SHARED_DIR
 
-for crawl_file in /home/joe/build/beagle/etc/beagle/crawl-*; do 
+for crawl_file in /etc/beagle/crawl-*; do 
     unset CRAWL_ENABLED CRAWL_DISABLE_FILTERING CRAWL_RECURSIVE CRAWL_CACHE_TEXT CRAWL_ALLOW_PATTERNS CRAWL_DENY_PATTERNS CRAWL_PATHS CRAWL_INDEX_NAME
 
     if [ -f $crawl_file ] ; then
@@ -67,7 +67,7 @@
                 IONICE="$IONICE -c 3"
             fi
 
-	    eval $IONICE su -s /bin/bash $CRAWL_USER -c \"MONO_SHARED_DIR=$MONO_SHARED_DIR /home/joe/build/beagle/sbin/beagle-build-index --target /home/joe/build/beagle/var/cache/beagle/indexes/$CRAWL_INDEX_NAME $OPTIONS $CRAWL_PATHS\" > /dev/null 2>&1
+	    eval $IONICE su -s /bin/bash $CRAWL_USER -c \"MONO_SHARED_DIR=$MONO_SHARED_DIR /usr/sbin/beagle-build-index --target /var/cache/beagle/indexes/$CRAWL_INDEX_NAME $OPTIONS $CRAWL_PATHS\" > /dev/null 2>&1
 	fi
     fi
 done
