diff -aurN beagle-0.1.3.orig/Best/best.desktop.in beagle-0.1.3/Best/best.desktop.in
--- beagle-0.1.3.orig/Best/best.desktop.in	2005-11-04 21:22:24.000000000 +0100
+++ beagle-0.1.3/Best/best.desktop.in	2005-12-08 08:21:04.000000000 +0100
@@ -1,9 +1,8 @@
 [Desktop Entry]
-X-SuSE-translate=true
 Encoding=UTF-8
 _Name=Beagle Search
 _GenericName=Desktop Search
 Exec=best --no-tray
 Icon=best
 Type=Application
-Categories=GTK;GNOME;Application;Core;Utility;Filesystem;
+Categories=GTK;GNOME;Utility;
diff -aurN beagle-0.1.3.orig/tools/beagle-settings.desktop.in beagle-0.1.3/tools/beagle-settings.desktop.in
--- beagle-0.1.3.orig/tools/beagle-settings.desktop.in	2005-11-04 21:22:25.000000000 +0100
+++ beagle-0.1.3/tools/beagle-settings.desktop.in	2005-12-08 08:22:06.000000000 +0100
@@ -1,10 +1,9 @@
 [Desktop Entry]
 Encoding=UTF-8
-X-SuSE-translate=true
 _Name=Search & Indexing
 _GenericName=Configure Desktop Search & Indexing
 _Comment=Configure search & indexing properties
 Exec=beagle-settings
 Icon=gnome-searchtool
 Type=Application
-Categories=Application;Settings;AdvancedSettings;Utility;Filesystem;X-Ximian-Settings;
+Categories=GTK;GNOME;Settings;AdvancedSettings;
